packages/react-native/src/Authenticator/Defaults/ConfirmResetPassword/ConfirmResetPassword.tsx

```diff
@@ -2,17 +2,15 @@ import React from 'react';
 import { Text } from 'react-native';
 
 import { ConfirmResetPasswordComponent } from './types';
+import { DefaultFooter } from '../../common/DefaultFooter';
```

Comment:
nit: Can we just do:

```jsx
import { DefaultFooter, DefaultHeader } from '../../common';
```

Reply:
I prefer to allow each export to be imported from its own module versus only through the main entry point (index file) as this reduces path resolution time.